### PR TITLE
Add missing canonical with "all=true" for post page

### PR DESCRIPTION
**Pull request by myokoym**

### CODE REVIEW

**Comment by myokoym** [comment]
I'll rebase.

**Comment by myokoym** [comment]
rebased.

File: test/integration/canonical_test.rb [hunk 1]
```diff
@@ -41,4 +41,19 @@ def test_normalize(data)
                    find('link[rel=canonical]', visible: false)[:href]
     end
   end
+
+  sub_test_case 'post page' do
+    data({
```
Comment (mtsmfm) [inline test/integration/canonical_test.rb:46]: TIL

**Comment by tricknotes** [comment]
looks shippable :shipit:

**Pull request merged**